src/ui/app/index.tsx

```diff
@@ -42,8 +42,12 @@ const App = () => {
       <Route path={`/${pathPrefix}/workflow/:id`} element={<RequireAuth user={user}><WorkflowPage user={user} /> </RequireAuth>} />
```

Comment:
added routes to access operator / artifacts using `DagID`.

Reply:
the :operatorId and :artifactId stuff makes sense to me here 👍